client/src/app/site/agenda/components/agenda-list/agenda-list.component.html

```diff
@@ -119,6 +119,11 @@ <h2>{{ 'Agenda' | translate }}</h2>
             <span>{{ 'Import' | translate }}</span>
         </button>
         <mat-divider></mat-divider>
+        <button mat-menu-item *osPerms="'agenda.can_manage'" class="red-warning-text" click="deleteAllSpeakersOfAllListsOfSpeakers()">
```

Comment:
it's `(click)="deleteAllSpeakersOfAllListsOfSpeakers()">`

Reply:
OMG :D I needed a break lol
Thanks